src/Windows/Avalonia.Win32/FramebufferManager.cs

```diff
@@ -25,7 +25,7 @@ public ILockedFramebuffer Lock()
             UnmanagedMethods.GetClientRect(_hwnd, out rc);
             var width = rc.right - rc.left;
             var height = rc.bottom - rc.top;
-            if (_fb == null || _fb.Width != width || _fb.Height != height)
+            if ((_fb == null || _fb.Width != width || _fb.Height != height) && width > 0 && height > 0)
```

Comment:
By doing this  you are canceling framebuffer resize. I think that you should cancel render pass completely, since window is not visible anyway.